generate.py

```diff
@@ -359,3 +363,116 @@ def normalize(code):
   link = f'https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes#{ base_code }' if len(base_code) == 2 else f'https://en.wikipedia.org/wiki/ISO_639:{ base_code }'
 
   print(f'[{ text }]({ link })')
+
+# Generate TMS files
+
+for tms in INTEGRATIONS:
+    tms_id = tms['id']
+    tms_name = tms['name']
+    tms_url = tms['tms_url']
+    tms_type = tms['type']
+    fuzzy_repair = tms.get('fuzzy_repair', False)
+    tms_open_source = tms.get('open_source', False)
+    tms_quality_estimation = tms.get('quality_estimation', False)
+    APIS_BY_ID = {api['id']: api for api in APIS + AGGREGATORS}
+    
+    tms_api_integrations = []
+    for integration in tms['api_integrations']:
+      try:
+        integration_data = {}
+        if type(integration) == type({}):
+          integration_slug = list(integration.keys())[0]
+          integration_data['slug'] = integration_slug
+          for k, v in integration[integration_slug].items():
+            integration_data[k] = v 
+        else:
+          integration_slug = integration
+          integration_data['slug'] = integration_slug
+        
+        integration_data['name'] = APIS_BY_ID[integration_slug]['name']
+
+        tms_api_integrations.append(integration_data)
+      except KeyError:
+        pass
+
+    if 'tms' in tms_type:
+      tms_type_description = 'translation management system'
+    if 'cat' in tms_type and 'tms' not in tms_type:
+      tms_type_description = 'computer-aided tool'
```

Comment:
We should use this - add it to `frontmatter` - then the template can have less logic.

But need to fix the logic a bit.

```
    _ = []
    if 'tms' in tms_type:
      _.append('translation management system')
    if 'cat' in tms_type and 'tms' not in tms_type:
      _.append('computer-assisted translation tool')
    tms_type_name = ' and '.join(types_names)
```